demo/midgard-sdk/src/scheduler.ts

```diff
@@ -1,10 +1,92 @@
-import { LucidEvolution, TxBuilder } from "@lucid-evolution/lucid";
+import { Data, LucidEvolution, TxBuilder, UTxO } from "@lucid-evolution/lucid";
+import { Effect } from "effect";
+import {
+  DataCoercionError,
+  getStateToken,
+  POSIXTimeSchema,
+  UnauthenticUtxoError,
+} from "./common.js";
```

Comment:
Again for consistency.
```suggestion
} from "@/common.js";
```